packages/py/python-aiohappyeyeballs/package.yml

```diff
@@ -0,0 +1,20 @@
+name       : python-aiohappyeyeballs
+version    : 2.4.3
+release    : 1
+source     :
+    - https://github.com/aio-libs/aiohappyeyeballs/archive/refs/tags/v2.4.3.tar.gz : 0bb4dacfcedb8ef80f8983d4f66242b926c4c29dd271d7f8178f9517ea66abf6
+homepage   : https://aiohappyeyeballs.readthedocs.io/
+license    : PSF-2.0
+component  : programming.python
+summary    : Happy Eyeballs for pre-resolved hosts
+description: |
+    This library exists to allow connecting with Happy Eyeballs (RFC 8305) when you already have a list of addrinfo and not a DNS name.
+builddeps  :
+    - python-build
+    - python-installer
+    - python-packaging
```

Comment:
If you are going for 2.4.4 `python-poetry-core` is needed.
`python-packaging` and `python-wheel` are not necessary.